components/FormCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react'
+import { Button } from './theme/Button'
+import { MdInput } from './MdInput'
+
+// capitalizes first letter of a string
+const capitalizeFirst = (str: string) => {
+  return str.replace(/./, char => char.toUpperCase())
+}
+type Option = {
+  title: string
+  placeHolder?: string
+  value?: string | number
+}
+
+type Btn = {
+  title: string
+  onClick?: Function
+}
+
+type FormCardProps = {
+  values: Option[]
+  buttons?: Btn[]
+  capitalizeTitle?: boolean
+  title?: string
+}
+
+type ButtonsProps = {
+  buttons: Btn[]
+  options: Option[]
+}
+
+type OptionsListProps = {
+  handleChange: (value: string, i: number) => void
+  capitalizeTitle: boolean
+  options: Option[]
+}
+
+const titleStyle: React.CSSProperties | undefined = {
+  textDecoration: 'underline',
+  marginBottom: 20
+}
+
+const formStyle: React.CSSProperties | undefined = {
+  textAlign: 'center',
+  padding: '10px',
+  border: '2px solid rgb(0,0,0,.3)',
+  backgroundColor: 'white'
+}
+
+const optionStyle: React.CSSProperties | undefined = {
+  border: '3px solid rgb(84, 64, 216, .3)',
+  padding: '10px',
+  backgroundColor: 'rgb(84, 64, 216, .04)',
+  textAlign: 'left'
+}
+
+const Buttons: React.FC<ButtonsProps> = ({ buttons, options }) => {
+  const btns = buttons.map((btn: Btn, i: number) => (
+    <div style={{ display: 'inline-block', margin: 10 }} key={i}>
+      <Button
+        onClick={() => btn.onClick && btn.onClick(options)}
+        type="success"
+      >
+        {btn.title}
+      </Button>
+    </div>
+  ))
+
+  return <>{btns}</>
+}
+
+export const OptionsList: React.FC<OptionsListProps> = ({
+  handleChange,
+  capitalizeTitle,
+  options
+}) => {
+  const inputs = options.map((option: Option, i: number) => {
+    const { title, value, placeHolder } = option
+    if (option.title === 'id') return []
+    const inputType =
+      title === 'description' ? (
```

Comment:
instead of hardcoding, could you set `if (option.type === MDINPUT)`

this component should export a few types.

Reply:
yes, i will do that. thanks!